soql-stdlib/src/main/scala/com/socrata/soql/functions/SoQLFunctions.scala

```diff
@@ -282,6 +282,15 @@ object SoQLFunctions {
   val MedianDisc = f("median_disc", FunctionName("median"), Map("a" -> (Ordered -- NumLike)), Seq(VariableType("a")), Seq.empty, VariableType("a"), isAggregate = true)(
     "Return a discrete median of a given set of numbers"
   )
+  val RegrIntercept = mf("regr_intercept", FunctionName("regr_intercept"), Seq(SoQLNumber, SoQLNumber), Seq.empty, SoQLNumber, isAggregate = true) (
+    "Return the OLS Regression intercept of (y, x) columns"
```

Comment:
Expand acronyms OLS and R

Reply:
👍  Expanded the acronyms (though r-squared is isn't an acronym, I clarified what it actually is)